App/VSTS/FollowButton.tsx

```diff
@@ -0,0 +1,66 @@
+/// <reference path="../../office.d.ts" />
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import { changeFollow } from '../actionsEZ';
```

Comment:
Every action should have a suffix as Action to make it more clear that you are invoking an action for this.